canvasapi/course.py

```diff
@@ -1411,8 +1411,14 @@ def get_multiple_submissions(self, **kwargs):
             :class:`canvasapi.submission.Submission`
         """
         if 'grouped' in kwargs:
-            warnings.warn('The `grouped` parameter must be empty. Removing kwarg `grouped`.')
-            del kwargs['grouped']
+            return PaginatedList(
```

Comment:
Instead of having separate returns, what about this?

```
cls = Submission

if 'grouped' in kwargs:
    cls = GroupedSubmission

return PaginatedList(
     cls,
     self._requester,
     'GET',
     'courses/{}/students/submissions'.format(self.id),
     {'course_id': self.id},
    _kwargs=combine_kwargs(**kwargs)
)
```